tests/backends/postgresql/test_returning.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+import unittest
+
+from django.db import router, connection
+from django.db.models.sql import InsertQuery
+from django.test import modify_settings, TestCase
+
+from ..models import Author, MTIReturning, ReturningModel
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL tests')
+@modify_settings(INSTALLED_APPS={'append': 'django.contrib.postgres'})
+class ReturningValuesTestCase(TestCase):
+    def test_pk_only(self):
+        db = router.db_for_write(Author)
+        query = InsertQuery(Author)
+        query.insert_values(Author._meta.fields, [], raw=False)
+        # this line will PostgreSQLModel an AttributeError without the accompanying fix
+        compiler = query.get_compiler(using=db)
+        compiler.return_id = True
+        sql = compiler.as_sql()[0][0]
+        self.assertIn('RETURNING "backends_author"."id"', sql)
+
+    def test_multiple_fields(self):
+        db = router.db_for_write(ReturningModel)
+        query = InsertQuery(ReturningModel)
+        query.insert_values(ReturningModel._meta.fields, [], raw=False)
+        # this line will PostgreSQLModel an AttributeError without the accompanying fix
+        compiler = query.get_compiler(using=db)
+        compiler.return_id = True
+        sql = compiler.as_sql()[0][0]
+        self.assertIn('RETURNING "backends_returningmodel"."id"'
+                      ', "backends_returningmodel"."created"', sql)
```

Comment:
Use hanging indent as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).